From fcffc09859c85ac179f0f87f17a57e97501c49da Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 21 Sep 2022 11:26:50 -0700
Subject: [PATCH] Announce overrides of content and version URLs

When using -v, -c, or -u to set alternate version or content URLs, print
out a message so it's more obvious.
---
 src/swupd_lib/globals.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/swupd_lib/globals.c b/src/swupd_lib/globals.c
index e2176d3a5615..32512ebe7550 100644
--- a/src/swupd_lib/globals.c
+++ b/src/swupd_lib/globals.c
@@ -533,6 +533,7 @@ static bool global_parse_opt(int opt, char *optarg)
 		}
 		return true;
 	case 'u':
+		print("Overriding version and content URLs with %s\n", optarg);
 		set_version_url(optarg);
 		set_content_url(optarg);
 		return true;
@@ -544,9 +545,11 @@ static bool global_parse_opt(int opt, char *optarg)
 		}
 		return true;
 	case 'c':
+		print("Overriding content URL with %s\n", optarg);
 		set_content_url(optarg);
 		return true;
 	case 'v':
+		print("Overriding version URL with %s\n", optarg);
 		set_version_url(optarg);
 		return true;
 	case 'F':
-- 
2.37.3

